src/components/pages/TDBytes/TDBytes.tsx

```diff
@@ -0,0 +1,211 @@
+import {
+  Box,
+  Center,
+  Flex,
+  Heading,
+  VStack,
+  Text,
+  Spacer,
+  Image,
+  Card,
+  CardBody,
+  CardHeader,
+  Button,
+  Divider,
+  Link,
+} from '@chakra-ui/react';
+import Footer from 'components/molecules/footer/Footer';
+import useTitle from 'hooks/useTitle';
+import tdbytesQR from 'assets/tdbytes-qr.png';
+import tdbytesMap from 'assets/tdbytes-location.png';
+import TextField from 'components/atoms/textfield/Textfield';
+import { Link as RouterLink, useHistory } from 'react-router-dom';
+import { useContext } from 'react';
+import { AuthenticateContext, Roles } from 'contexts/authProvider';
+import useForm from 'hooks/useForm';
+import { useToast } from 'hooks/useToast';
+import { ProductSuggestionPayload } from 'models/apiModels';
+import { addSuggestion } from 'api/kiosk';
+
+const TDBytesPage = () => {
+  useTitle('TD Bytes');
+  const history = useHistory();
+  const { authenticated, role } = useContext(AuthenticateContext);
+  const { addToast } = useToast();
+
+  const hasAuthorization = role === Roles.admin || role === Roles.kiosk_admin;
+
+  const moveToLoginPage = () => {
+    history.push('/login');
+  };
```

Comment:
if the suggestion route is moved to a privet route this can be removed as a the privateRoute sends an unauthorised user to login before redirecting the back to the desired destination. So redirecting to `/suggestion` automatically sends the user to login if not authenticated

Reply:
I'm thinking the TDBytes page should be viewable to anyone, also if they aren't logged in. Posting suggestions however should require login.